src/OpenTelemetry.Contrib.Instrumentation.AWS/Implementation/AWSTracingPipelineCustomizer.cs

```diff
@@ -0,0 +1,50 @@
+﻿// <copyright file="AWSTracingPipelineCustomizer.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using Amazon.Runtime;
+using Amazon.Runtime.Internal;
+
+namespace OpenTelemetry.Contrib.Instrumentation.AWS.Implementation
+{
+    internal class AWSTracingPipelineCustomizer : IRuntimePipelineCustomizer
+    {
+        private readonly AWSClientInstrumentationOptions options;
+
+        public AWSTracingPipelineCustomizer(AWSClientInstrumentationOptions options)
+        {
+            this.options = options;
+        }
+
+        public string UniqueName
```

Comment:
i'm not sure, but since u created consts in a few files above, should we move to consts as well? ignore if u think that is irrelevant.

Reply:
I created the constants file primarily for the Semantic Conventions which I expect to grow/change in the future as we add more custom AWS services and properties to record. For the customizer name here I don't expect it to ever change or exposed to a user so I think we can leave it here as is.